qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AllergyPage.java

```diff
@@ -23,6 +23,8 @@
 	private static final By ALLERGY_STATUS = By.cssSelector(".allergyStatus");
 	private static final By ADD_NO_KNOWN = By.xpath("//*[contains(text(),'No Known Allergy')]");
 	private static final By DELETE_NO_KNOWN = By.className("delete-action");
+	private static final By RETURN_BUTTON = By.cssSelector("#content > button.cancel");
+	
```

Comment:
any reason for adding an empty line here?

Reply:
> any reason for adding an empty line here?

Oh I had not seen this comment. Let me fix it @kdaud